react/src/base/inputs/DatePicker.stories.js

```diff
@@ -69,56 +86,68 @@ export const invalidDatePicker = () => (
 invalidDatePicker.story = {
   name: 'Invalid',
   parameters: {
-    jest: ['SprkErrorContainer', 'SprkInputContainer'],
+    jest: [
+      'SprkErrorContainer',
+      'SprkInputContainer',
+      'SprkDatePicker',
+      'SprkLabel',
+      'SprkIcon',
+    ],
   },
 };
 
 export const disabledDatePicker = () => (
   <SprkInputContainer>
-    <SprkDatePickerInput
-      name="date"
-      placeholder="01/01/2019"
-      errorMessage="Incorrect date."
-      label="Date"
-      disabled
-    />
+    <div className="sprk-b-InputContainer__icon-container">
+      <SprkLabel isDisabled>Date</SprkLabel>
+      <SprkIcon
+        iconName="calendar"
+        additionalClasses="
+          sprk-c-Icon--stroke-current-color sprk-b-InputContainer__icon"
+      />
+      <SprkDatePicker
+        isDisabled
+        placeholder="01/01/2021"
+        additionalClasses="sprk-b-TextInput--has-svg-icon sprk-u-Width-100"
+      />
+    </div>
   </SprkInputContainer>
 );
 
 disabledDatePicker.story = {
   name: 'Disabled',
   parameters: {
-    jest: ['SprkInputContainer'],
+    jest: ['SprkInputContainer', 'SprkDatePicker', 'SprkLabel', 'SprkIcon'],
   },
 };
 
 export const hugeDatePicker = () => (
```

Comment:
These Huge stories don't have icons intentionally. We don't have style for Huge inputs with icons yet.